src/pages/Bridge/PoolInfo.tsx

```diff
@@ -0,0 +1,41 @@
+import { ChainId } from '@kyberswap/ks-sdk-core'
+import { Trans } from '@lingui/macro'
+import { Flex, Text } from 'rebass'
+
+import { NETWORKS_INFO } from 'constants/networks'
+import useTheme from 'hooks/useTheme'
+
+import { formatPoolValue } from './helpers'
+import { MultiChainTokenInfo } from './type'
+
+const PoolInfo = ({
+  chainId,
```

Comment:
Why do we have to pass chainId as `props`? Remove it and get directly from `useActiveWeb3React`

Reply:
saw `chainId={chainIdOut}`

Reply:
chainId here not only cur chain
<img width="189" alt="image" src="https://user-images.githubusercontent.com/33005392/198190920-87117d2d-e3ef-4cf8-bb94-a62b49a04a79.png">